_infra/helm/locust/locustfiles/requests.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
This will probably need to be a configmap object rather than a raw json file. Something like (I'll leave the detail to you)
```
apiVersion: v1
kind: ConfigMap
metadata:
  name: requests.json
data:
  requests.json: |
   {
    "requests": [
        {
            "method": "GET",
            "url": "/surveys/todo",
            "expected_response_text": "Click on the survey name to complete your questionnaire"
        },
        {
            "method": "GET",
            "url": "/surveys/history",
            "expected_response_text": "<title>Completed surveys | ONS Business Surveys</title>"
        },
        {
            "method": "GET",
            "url": "/secure-message/threads",
            "expected_response_text": "<title>Messages | ONS Business Surveys</title>"
        }
    ]
}
```

Which is then available as a mounted resource.